src/ui/assets/AssetsList.js

```diff
@@ -384,6 +385,7 @@ AssetsList.propTypes = {
   perPageOptions: PropTypes.arrayOf(PropTypes.number),
   onAssetSelected: PropTypes.func.isRequired,
   onClickDelete: PropTypes.func.isRequired,
+  onResetFilterinCategories: PropTypes.func.isRequired,
```

Comment:
It seems that there's a minor typo.
```suggestion
  onResetFilteringCategories: PropTypes.func.isRequired,
```